app/models/login_redirect_application.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require 'common/models/redis_store'
+
+# Users logging in via SSOe may need to be redirected back to an external
+# application after authentication rather than the VA.gov home page.  When a
+# authentication request comes in, with the necessary parameter, we can use
+# this Redis namespace to temporarily store a value so that when a matching
+# SAML response comes back from SSOe we know where to redirect the newly
+# authenticated user.
+class LoginRedirectApplication < Common::RedisStore
```

Comment:
I really like this and think we could use this to do a better job of tracking login metrics (timing, abandon rate), so I'm wondering if we could give this a more generic name so that we could use it for other login-related attributes in the future.